boot/bootutil/src/encrypted.c

```diff
@@ -513,6 +513,13 @@ boot_enc_decrypt(const uint8_t *buf, uint8_t *enckey)
 #endif
     int rc = -1;
 
+#if defined(MCUBOOT_HW_KEY)
+    rc = boot_retrieve_private_key(&bootutil_enc_key.key, &bootutil_enc_key.len);
+    if (rc) {
+	return rc;
```

Comment:
Make sure your editor is configured to insert spaces instead of tabs when working on mcuboot source. In this case, this should be 8 spaces rather than a tab.

Reply:
done